cmd/bridge/config/config.toml

```diff
@@ -3,7 +3,8 @@
     MultisigContractAddress = "3009d97FfeD62E57d444e552A9eDF9Ee6Bc8644c" # the eth address for the bridge contract
     SafeContractAddress = "A6504Cc508889bbDBd4B748aFf6EA6b5D0d2684c"
     PrivateKeyFile = "keys/ethereum.sk" # the path to the file containing the relayer eth private key
-    GasLimit = 500000
+    GasLimitBase = 50000
```

Comment:
do not forget to change the configs from the config repo also
Also, is this enough? From the old configs, we had 500k. Now, for 10 transfers we have 10*30k+50k = 350k.
This gas cost is also proportional with the quorum size.

Reply:
GasLimitBase increased to 20000, so we have the same "starting" value